package.json

```diff
@@ -955,7 +954,7 @@
     "url": "git+https://github.com/ansible/vscode-ansible.git"
   },
   "scripts": {
-    "compile": "tsc -b && copyfiles -u 4 'node_modules/@ansible/ansible-language-server/out/server/**/*' out",
+    "compile": "tsc -b && tsc -p ./packages/ansible-language-server/tsconfig.json --outDir ./out/server --sourceMap",
```

Comment:
Replaced `copyfiles` with TypeScript compiler + outDir & sourceMap option. By doing this, generated source maps correctly point to actual source location under ./package/ansible-language-server/